storage/innobase/xtrabackup/test/t/pxb-2429.sh

```diff
@@ -0,0 +1,32 @@
+# PXB-2349 crash in PXB during tablespace encryption
+if ! is_debug_server  ; then
+	skip_test "Required debug version"
+fi
+
+. inc/keyring_file.sh
+start_server
+
+mysql -e "set global innodb_page_cleaner_disabled_debug=on" test 2>/dev/null >/dev/null
+mysql -e "set global innodb_checkpoint_disabled=on" test 2>/dev/null >/dev/null
+mysql -e "create tablespace ts add datafile 'ts.ibd'" test 2>/dev/null >/dev/null
+
+mkdir $topdir/backup
+
+vlog "Case#1 backup just after normal create"
+xtrabackup --backup --target-dir=$topdir/backup
+
+vlog "Case#2 backup after encryption=y"
+mysql -e "alter tablespace ts encryption='y'" test 2>/dev/null >/dev/null
+vlog "alter executed succesful"
+mkdir $topdir/backup2
+xtrabackup --backup --target-dir=$topdir/backup2
+
+vlog "Case#3 backup after encryption=N"
+mysql -e "alter tablespace ts encryption='N'" test 2>/dev/null >/dev/null
+mkdir $topdir/backup3
+xtrabackup --backup --target-dir=$topdir/backup3
+
+vlog "Case#4 backup after encryption=Y"
+mysql -e "alter tablespace ts encryption='Y'" test 2>/dev/null >/dev/null
+mkdir $topdir/backup4
+xtrabackup --backup --target-dir=$topdir/backup4
```

Comment:
backup dirs are deleted automatically? if not, please delete them

Reply:
also please paste result of the testcase as github comment

Reply:
./run.sh -t t/pxb-2429.sh
Using /Users/rahulmalik/MySQL/src/x8/bld/storage/innobase/xtrabackup/test/var as test root
Detecting server version...
Running against MySQL 8.0.23-debug (InnoDB 8.0.23)
Using 'xtrabackup' as xtrabackup binary
Using 8 parallel workers

==============================================================================
TEST                                   WORKER    RESULT     TIME(s) or COMMENT
------------------------------------------------------------------------------
pxb-2429                                 w1	[passed]    34
==============================================================================
Spent 34 of 56 seconds executing testcases

SUMMARY: 1 run, 1 successful, 0 skipped, 0 failed

See results/ for detailed output

Reply:
whole backup directory is cleaned after the backup.